src/sentry/api/serializers/models/organization.py

```diff
@@ -118,30 +118,49 @@ class DetailedOrganizationSerializer(OrganizationSerializer):
     def get_attrs(self, item_list, user, **kwargs):
         return super(DetailedOrganizationSerializer, self).get_attrs(item_list, user)
 
-    def serialize(self, obj, attrs, user, access):
-        from sentry import experiments
-        from sentry.api.serializers.models.project import ProjectSummarySerializer
-        from sentry.api.serializers.models.team import TeamSerializer
+    def _project_list(self, organization, access):
+        member_project_ids = []
+        member_projects = []
+        for project in access.projects:
```

Comment:
Does it make sense to just filter down `access.projects` to not include deleted projects? Or is there some way that users are able to undelete a project that they need this permission for?

Reply:
Filtering down access.projects seems reasonable to me, there isn't currently an API endpoint to undelete a project. I'd like to do that as a follow up pull request as this issue is causing some users pain and changing `access` is generally higher risk.